latest/panel/building/build.md

```diff
@@ -0,0 +1,31 @@
+# Building panel assets
+
+!> Please note that we dont provide support to modified installs
```

Comment:
!> Please note that support can't be provided in the Jexactyl Discord if you choose to modify the Panel. You are doing this at your own risk.